contracts/src/lido/README.md

```diff
@@ -0,0 +1,105 @@
+# Lido's Scroll Bridge
+
+The document details the implementation of the bridging of the ERC20 compatible tokens[^*] between Ethereum and Scroll chains.
+
+It's the first step of Lido's integration into the Scroll protocol. The main goal of the current implementation is to be the strong foundation for the long-term goals of the Lido expansion in the Scroll chain. The long-run picture of the Lido's integration into L2s includes:
+
+- Bridging of Lido's tokens from L1 to L2 chains
+- Instant ETH staking on L2 chains with receiving stETH/wstETH on the corresponding L2 immediately
+- Keeping UX on L2 as close as possible to the UX on Ethereum mainnet
+
+At this point, the implementation must provide a scalable and reliable solution for Lido to bridge ERC20 compatible tokens between Scroll and the Ethereum chain.
+
+[^*]: The current implementation might not support the non-standard functionality of the ERC20 tokens. For example, rebasable tokens or tokens with transfers fee will work incorrectly. In case your token implements some non-typical ERC20 logic, make sure it is compatible with the bridge before usage.
+
+## Security surface overview
+
+| Statement                                                                                                                                    | Answer |
+| -------------------------------------------------------------------------------------------------------------------------------------------- | ------ |
+| It is possible to bridge wstETH forth and back using this bridge                                                                             | Yes    |
+| The bridge using a canonical mechanism for message/value passing                                                                             | Yes    |
+| The bridge is upgradeable                                                                                                                    | Yes    |
+| Upgrade authority for the bridge                                                                                                             | TBA    |
+| Emergency pause/cancel mechanisms and their authorities                                                                                      | TBA    |
+| The bridged token support permits and ERC-1271                                                                                               | Yes    |
+| Are the following things in the scope of this bridge deployment:                                                                             |        |
+| - Passing the (w)stETH/USD price feed                                                                                                        | No     |
+| - Passing Lido DAO governance decisions                                                                                                      | TBA    |
```

Comment:
:warning:  I am afraid we would need to have this part for the DAO recognition :pray: 
e.g., for Linea have the following: https://github.com/Consensys/governance-crosschain-bridges/blob/linea-onboarding/contracts/bridges/LineaBridgeExecutor.sol

They derived the following reference impl: https://github.com/lidofinance/governance-crosschain-bridges